README.md

```diff
@@ -11,8 +11,7 @@ focused on development & contributing documentation.
 ## Contributing to this repository
 
 We want to make contributing to this repository to be as easy as possible
```

Comment:
Nit...
"We want to make contributing to this repository as easy as possible"

Reply:
Updated